In Python 2.7.9, SSL certificate validation is enabled by default. This is a problem
for Splunk as we ship with a self-signed certificate. This patch merely sets it
back to the previous behavior. You can read more here:

https://www.python.org/dev/peps/pep-0476/#opting-out

/Users/ineeman/work/git/main2/contrib/Python-2.7.9/Lib/ssl.py
diff --git a/tmp/ssl.py.orig b/tmp/ssl.py
index 0f822277e96e..ca7a24c35f33 100644
--- a/Lib/ssl.py.orig
+++ b/Lib/ssl.py

@@ -469,7 +469,7 @@ def _create_unverified_context(protocol=PROTOCOL_SSLv23, cert_reqs=None,
     return context
 
 # Used by http.client if no context is explicitly passed.
-_create_default_https_context = create_default_context
+_create_default_https_context = _create_unverified_context
 
 
 # Backwards compatibility alias, even though it's not a public name.
